src/System.Net.Http/src/System/Net/Http/Managed/HttpConnectionPool.cs

```diff
@@ -351,6 +365,33 @@ public bool CleanCacheAndDisposeIfUnused()
             return false;
         }
 
+        public override string ToString() => $"{nameof(HttpConnectionPool)}(Connections:{_associatedConnectionCount})"; // Description for diagnostic purposes
+
+        private void Trace(string message, [CallerMemberName] string callerName = null)
```

Comment:
I could do that.  I've avoided the main costs involved in evaluating parameters in this pattern as well, but I can change it.